engine/script/src/script.cpp

```diff
@@ -872,6 +872,14 @@ namespace dmScript
         return false;
     }
 
+    bool CheckURL(lua_State* L, dmMessage::URL* out_url) {
+        bool result = GetURL(L, out_url);
+        if (!result) {
+            luaL_error(L, "no URL could be found in the current script environment");
```

Comment:
The recommendation from Lua is to always do a return: "return luaL_error()"

Also type: "no" -> "No"